### DISCUSSION TITLE
It doesn't work as well as before

### DISCUSSION THREAD

**Discussion by unecc80** [Andre0512/hon#94] [General]
Hello, I am writing because I have been using the hon addon for about 1 month and it has always worked very well.
For a few days, however, it has been telling me that the entities of the washing machine are not available with this version.

washing machine RO14126DWMST-S

The question is: how can I downgrade with the version that worked perfectly for me?

**Comment by Andre0512** [comment]
You can downgrade with HACS:

![Screenshot from 2023-07-19 00-00-56](https://github.com/Andre0512/hon/assets/20325077/9da8dc85-bd13-49f1-ab6b-20aaf546a2e9)

or use git and checkout the tag of the wanted version.

**Comment by unecc80** [comment]
Hi done in past.
 it's the same.
I've try 9.0, old versions and also new beta versions.
Same problem. Entities are disappeared
